internal/config/config.go

```diff
@@ -86,10 +85,23 @@ type Config struct {
 var regionalConfig = map[string]RegionalConfig{
 	"AUT": {
 		Models: map[string]Model{
-			"2060": {
+			"2060S": {
 				SKU: strPtr("5394902900"),
 			},
-			"3070": {},
+			"2070S": {
```

Comment:
I would prefer we just kept to the 2060 for now as it's in stock pretty much globally adding too many flags right now would confuse users.

Reply:
Also changing what was previously working 2060 to 2060S would confuse users, probably should avoid making any breaking changes to the existing flags/input when possible.

Reply:
ok on the 2060 thing, but can we add the new models to a section near the bottom called older generation cards?

Reply:
I mean I'm okay with adding older models I guess but I want to be super consistent about the naming convention for those models, I'm not sure what S indicates? 

Like 2060, 2070, 2080, 2080TI, (I don't really like adding Titan as it's a totally different naming convention)

It might also make sense adding like a -list option flag so you could do like -region=USA -list to list models or something.

Reply:
But if we explicitly say what each model is it won't be a problem. I second doing some kind of -list flag. And the S is Super the only models they carry now.

Reply:
I don't know how to make a new flag other than what I just did. Please help @ianmarmour

Reply:
So I hear you regarding the SUPER for S but I do wonder if this is a case where we should do what users want not what they say. If super is the only model for sale maybe we support super without the flag, Titan cards I guess I can give way on that. My biggest thought here is I want to write code that is simple that I don't have to support long term, I.E. when NVIDIA decides to call them GGGTTTTXXXX SUPER MEGA ULTRA I can still just put 4080 lol.

Reply:
true

Reply:
I changed the lineup to 2060, 2070, 2080, 2080TI, 3080, and 3090 only now